sanic/request.py

```diff
@@ -426,6 +430,19 @@ def credentials(self) -> Optional[Credentials]:
                 pass
         return self.parsed_credentials
 
+    @property
+    def keep_blank_values(self) -> bool:
+        """Getter for preserving blank form values"""
+        return self._keep_blank_form_values
+
+    @keep_blank_values.setter
+    def keep_blank_values(self, keep_blank_values: bool = False):
+        """Setter for preserving blank form values
+
+        Defaults to false
+        """
+        self._keep_blank_form_values = keep_blank_values
+
```

Comment:
What I meant was that the setter methods weren't really necessary, and we could just make that property public like you've now done, so I guess these methods are no longer necessary.

```suggestion
```

Reply:
discussed on discord